drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -27,6 +27,43 @@ hideRightPanel=Hide Right Panel
 testTools=Test Tools
 testEditor=Test Editor
 scenarioCheatSheet=Scenario Cheatsheet
+dmnCheatSheet=<p><h4>DMN-based scenarios</h4>\
+  <p>DMN-based scenarios use the FEEL expression language. Supported data types are:</p>\
```

Comment:
Before list the supported data types please add a paragraph with
`The text in a GIVEN cell is evaluated as literal expression and in the EXPECT as unary expression. This is because it verifies if the unary test is valid using the result from DMN evaluation`
or something similar

Reply:
I would rephrase it to the following:

> Values in the GIVEN part of the table are evaluated as literal expressions, whereas values in the EXPECT section are evaluated as unary expressions. The correctness of these unary tests is verified using the results from a DMN evaluation.

Reply:
this is definitely not recommended way. the i18n mechanism is not meant to have HTML on it.

it may break and eventually will break.